modules/ROOT/pages/migrate-ah.adoc

```diff
@@ -0,0 +1,42 @@
+:experimental:
+
+= Migrating from Fedora Atomic Host (FAH) to Fedora CoreOS (FCOS)
+
+== Overview
+
+https://www.projectatomic.io/[Fedora Atomic Host] was a system to deploy applications in containers. Existing FAH users are encouraged to migrate to FCOS, as the project is reaching its end-of-life.
+
+FAH used `cloud-init` for provisioning, which required users to provide a `cloud-config` file as userdata for configuration of the instance. Since FCOS Ignition and `cloud-init` are different and have overlapping feature sets, it is not trivial to convert `cloud-init` files to Ignition. Currently, there is no tool for this conversion, so you must manually provision FCOS instances with Ignition. Refer to xref:fcct-config.adoc[FCCT Specification] for an explanation of the available configuration options.
```

Comment:
Maybe instead of "so you must manually provision FCOS instances with Ignition.", something like "so you must manually convert cloud-init configs to Fedora CoreOS configs." ?

Reply:
Thanks! I was struggling with this one.